src/main/resources/application.yaml

```diff
@@ -36,6 +36,8 @@ elasticsearch:
 deletable.case.types: ${DELETE_CASE_TYPES:}
 simulated.case.types: ${SIMULATED_CASE_TYPES:}
 
+request.limit: ${CCD_DISPOSER_REQUEST_LIMIT:100}
```

Comment:
set the default limit to 1000 please

Reply:
Please add integration tests too.

Reply:
Limit has been changed to 1000, I'll add Integration test